devenv/main.py

```diff
@@ -84,6 +85,9 @@ def devenv(argv: Sequence[str], config_path: str) -> ExitCode:
 
 
 def main() -> ExitCode:
+    if user == "root":
```

Comment:
this is for https://github.com/getsentry/terraform-sandboxes.private/pull/83#issuecomment-2221498154